crates/mcp/src/surfpool/start_surfnet.rs

```diff
@@ -10,10 +12,34 @@ pub struct StartSurfnetResponse {
 
 #[derive(Serialize)]
 pub struct StartSurfnetSuccess {
-    pub surfnet_url: String,
+    pub kind: StartSurfnetKind,
+    pub surfnet_url: Option<String>,
```

Comment:
Is there a reason for this to be optional? Seems like we always provide it when creating this struct